lib/usage.js

```diff
@@ -122,9 +134,10 @@ module.exports = function (yargs, y18n) {
         return acc
       }, {})
     )
+    var theWrap = getWrap()
```

Comment:
Ah jeez yeah, I see that now. OK, second arg is not too bad then! And no prob with the nitpicks; maintenance costs matter. :)